ui/app/AppLayouts/Chat/ChatLayout.qml

```diff
@@ -81,10 +81,11 @@ StatusAppThreePanelLayout {
     }
 
     showRightPanel: {
-        if (!localAccountSensitiveSettings.showOnlineUsers || !localAccountSensitiveSettings.expandUsersList) {
+        let chatContentModule = root.rootStore.currentChatContentModule()
+        if (!localAccountSensitiveSettings.showOnlineUsers || !localAccountSensitiveSettings.expandUsersList
+                || (chatContentModule && chatContentModule.chatDetails.type === Constants.chatType.publicChat)) {
```

Comment:
Just a suggestion.. Could we manage `chatContentModule` conditions in a separate `if` condition maybe grouping it with the next condition in line 88/89?

Reply:
Done, thanks @noeliaSD for the suggestion